.github/workflows/coding-standards.yml

```diff
@@ -0,0 +1,48 @@
+name: Coding Standards
```

Comment:
The previous `linting.yml` file was doing more than this one (checking PHP compatibility, which was currently set to 7.4 but was going to be expanded to 8.0 and eventually 8.1) plus running phpcs, and using cache. I'd recommend removing this file and keeping the other one.

Reply:
PHP Compatibility is set up on the Alley interactive ruleset. And caching is handled by default, see https://github.com/ramsey/composer-install#usage

Reply:
See here: https://github.com/alleyinteractive/alley-coding-standards/blob/main/Alley-Interactive/ruleset.xml#L25

We just need to set our desired compatibility version: `<config name="testVersion" value="7.4-"/>`

Reply:
Great to hear that caching is enabled by default. I can likely simplify a few of my other repos that use GitHub actions.

For the PHP Compatibility sniff, I'd prefer that be broken out. We need to check against multiple versions of PHP, but we only need to run the "normal" phpcs sniffs in the Alley ruleset once. To optimize efficiency here, we can run the full phpcs suite on one version of PHP (let's say 7.4 for now) and then we can run the PHP compatibility checks against a matrix of PHP versions that we want to support. That will enable us to check for compatibility against the PHP versions that we are supporting without running the full phpcs ruleset more than once.

Reply:
@kevinfodness 

> Great to hear that caching is enabled by default. I can likely simplify a few of my other repos that use GitHub actions.

Just a quick FYI: The action I'm using differs from the one I saw you and other folks using.

Correct me if I'm wrong, but it is my understanding that `<config name="testVersion" value="7.4-"/>` will test PHP compatibility from PHP 7.4+, or 5.6+, etc. Meaning that PHP 8, etc, is also being tested here.

> checks against a matrix of PHP versions that we want to support

If that's correct, why is this necessary? Isn't that already bundled into the Alley Interactive ruleset? 

`composer phpcs -- --standard=PHPCompatibility --extensions=php --runtime-set testVersion ${{ matrix.php }} --ignore="/vendor/*"
`

I'm under the assumption that I'm misunderstanding how the `testVersion` config is working with PHPCS.

Reply:
Actually, it turns out that _I_ was the one who was misunderstanding how `testVersion` works - I didn't realize that you could set a range, or leave it open-ended. This is great. Thanks!